src/views/WriteView.vue

```diff
@@ -68,16 +65,22 @@ export default {
             resetHeight(this.$refs.msgContainer);
         },
         spark() {
-            this.messages.push(
-                db.newMsg(this.$refs.typeWriterContainer.innerText),
-            );
+            if (this.msgtoEdit == null) {
```

Comment:
I would suggest using `msgToEdit`, and perhaps latter we will add choice to either update the original content or create a new message. Like if use `shift+enter` will update the content, and use `ctrl+enter` to create a new message.